osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -115,18 +117,18 @@ public DrawableStoryboardAnimation(StoryboardAnimation animation)
         }
 
         [BackgroundDependencyLoader]
-        private void load(IBindable<WorkingBeatmap> beatmap, TextureStore textureStore)
+        private void load(IBindable<WorkingBeatmap> beatmap, TextureStore textureStore, Storyboard storyboard)
         {
             for (var frame = 0; frame < Animation.FrameCount; frame++)
             {
                 var framePath = Animation.Path.Replace(".", frame + ".");
 
-                var path = beatmap.Value.BeatmapSetInfo.Files.Find(f => f.Filename.Equals(framePath, StringComparison.OrdinalIgnoreCase))?.FileInfo.StoragePath;
-                if (path == null)
```

Comment:
does this work as expected without this check? kinda undefined behaviour now.. (the case where `UseSkinSprites == false`)

Reply:
Good point. While setting a null texture on a sprite is supported as far as I can tell it should probably not happen if it can be avoided.

Reply:
Yeah it does look to work. Maybe we do want to initialise the sprites regardless thinking forward to storyboard editor implementation? They look to show up in the draw visualiser and behave as normal (but not display due to missing texture).

I think it's fine to leave this way since it doesn't crash or anything.

Reply:
Actually never mind, will change as i think it improves readability (and what I said doesn't make much sense – this is already encapsulated in a drawable representing the storyboard sprite).

Reply:
Ended up combining the lookup logic. Feels quite a bit cleaner.